### ISSUE TITLE
[🐛 Bug]: Chrome 106 won't start from LTS nodejs

### ISSUE THREAD

**Issue by 4aiman** [SeleniumHQ/selenium#11154]
### What happened?

Using latest Selenium Webdriver 4.5.0.
My tests suddenly stopped working. 
Upon investigation I found the #10970 issue which seemed like my case but has been fixed in the latest release.
I've tried to reinstall nodejs, Chrome, and all related modules to no luck.

I've tried to use a manually downloaded chromedriver and adding it to PATH instaed of using npm packaged chromedriver.

Setup with chromedriver npm package:
```
const {Builder} = require('selenium-webdriver');
require("chromedriver");

(async function helloSelenium() {
    let driver = await new Builder().forBrowser('chrome').build();
    await driver.get('https://selenium.dev');
    await driver.quit();
})();
```

Setup with chromedriver in PATH:
```
(async function helloSelenium() {
	let browser = require('selenium-webdriver/chrome')
	let service = await new browser.ServiceBuilder().setPort("45454").build()
	let options = await new browser.Options()
	let driver = await browser.Driver.createSession(options, service);
	driver.get("https://google.com")
})();
```

Both don't work, both fail with:
```
Timed out waiting for the WebDriver server at http://127.0.0.1:<port number>/
```

### How can we reproduce the issue?

```shell
I've posted main.js, versions of node, npm and related packages.
I believe I've mentioned everything relevant.
For comparison, the python code works with the same chromedriver.exe nodejs code fails to.

Can reproduce with firefox as well.
```

### Relevant log output

```shell
>(Get-WMIObject win32_operatingsystem) | Select Version
Version
-------
10.0.22621

>node -v
v16.18.0

>npm -v
8.19.2
>cat main.js
const {Builder} = require('selenium-webdriver');
require("chromedriver");
(async function helloSelenium() {
    let driver = await new Builder().forBrowser('chrome').build();
    await driver.get('https://selenium.dev');
    await driver.quit();
})();

>cat package.json
{
  "name": "pytest",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "chromedriver": "^106.0.1",
    "selenium-webdriver": "^4.5.0"
  }
}

>npm list
+-- chromedriver@106.0.1
`-- selenium-webdriver@4.5.0

>node main.js
<path_to_current_dir>\node_modules\selenium-webdriver\http\util.js:88
        reject(Error('Timed out waiting for the WebDriver server at ' + url))
               ^

Error: Timed out waiting for the WebDriver server at http://127.0.0.1:57729/
    at onError (<path_to_current_dir>\node_modules\selenium-webdriver\http\util.js:88:16)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)


>cat test.py
from selenium import webdriver
from selenium.webdriver.chrome.options import Options
chrome_options = Options()
chrome_options.add_argument('--no-sandbox')
driver = webdriver.Chrome('chromedriver.exe', options=chrome_options)
driver.get('http://www.google.com')
print('test')
driver.close()

>python3 test.py
test.py:5: DeprecationWarning: executable_path has been deprecated, please pass in a Service object
  driver = webdriver.Chrome('chromedriver.exe', options=chrome_options)

DevTools listening on ws://127.0.0.1:53772/devtools/browser/3f089046-8304-4369-b3ad-7b744a1ea156
test
```

### Operating System

Windows 11

### Selenium version

4.5.0

### What are the browser(s) and version(s) where you see this issue?

Chrome 106

### What are the browser driver(s) and version(s) where you see this issue?

ChromeDriver 106.0.5249.61

### Are you using Selenium Grid?

no

**Comment by 4aiman** [comment]
Rolling back to 4.4.0 solved my issues
```npm install selenium-webdriver@4.4.0```

**Comment by TamsilAmani** [comment]
@4aiman I was going through this issue and everything is working for me as expected. Here is what I did...

**REQUIREMENTS:**
```
node --version             
v16.18.0

package.json
"dependencies": {
    "chromedriver": "^106.0.1",
    "selenium-webdriver": "^4.5.0"
}

PATH to chromedriver = /opt/homebrew/bin/chromedriver
```

**1. Setup with chromedriver npm package:**
This works fine. I used your code as it is without any change. Used `npm install` to install the dependencies (chromedriver and selenium) before running `node main.js`

**2. Setup with chromedriver in PATH:**
This worked fine with a little tweak. You need to use `await` with `driver.get()`. The following code works fine.

```
(async function helloSelenium() {
	let browser = require('selenium-webdriver/chrome')
	let service = await new browser.ServiceBuilder().setPort("45454").build()
	let options = await new browser.Options()
	let driver = await browser.Driver.createSession(options, service);
        await driver.get("https://google.com")
        await driver.quit()
})();
```

Can you create a new project and install the dependencies from scratch? I'm using MacOS and not Windows. Check if your PATH to chromedriver is correct and working.

**Comment by 4aiman** [comment]
@TamsilAmani 
I don't really want to mess with node installation again since it's finally working now after 2 days of trying to figure this one out.
I'm on current nodejs v19 now and 4.4.0 selenium-webdriver.
4.5.0 doesn't work for me specifically in nodejs, regardless of how I use chromiumdriver.
Also, judging by the path, you're not on Windows 11.

What I can do is test this 
```
(async function helloSelenium() {
	let browser = require('selenium-webdriver/chrome')
	let service = await new browser.ServiceBuilder().setPort("45454").build()
	let options = await new browser.Options()
	let driver = await browser.Driver.createSession(options, service);
        await driver.get("https://google.com")
        await driver.quit()
})();
```
and this 
```
{
  "name": "pytest",
  "version": "1.0.0",
  "description": "",
  "main": "main.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
	  "dependencies": {
		"chromedriver": "^106.0.1",
		"selenium-webdriver": "^4.5.0"
	}
}
```
In a new folder and confirm it's not working, giving me the same timeout.

Then I downgrade selenium-webdriver to 4.4.0 and it works just fine.
I'm at a loss. At first I thought the latest windows 11 update somehow screwed the environment, but python examples work just fine.

**Comment by 4aiman** [comment]
I have no idea what has happened, but the 4.4.0 stopped working too.
I've spent a lot of time trying different versions of node, working around some weird certificate issues and node-gyp trying to build something when and where it shouldn't really. 
I even went ahead and tried on a separate OS and a separate device. All in vain.

Spent three days only to realize it was an *issue with my ISP* all along :|

In the end I've used my mobile phone to create a hotspot and all issues magically disappeared. On both devices and two OSes.
I still don't know why python continued to work flawlessly with selenium all this time.

So sorry for bothering you all.
Hope next time I'll actually do something useful.
